.eslintrc.js

```diff
@@ -14,15 +14,15 @@ module.exports = {
     ecmaVersion: 2018,
   },
   rules: {
-    'prettier/prettier': 'allow',
-    'import/no-extraneous-dependencies': 'allow',
+    'prettier/prettier': 1,
```

Comment:
I was receiving errors from prettier saying that `allow` was deprecated, so I updated these configurations to match their new convention.